webapp/app/forms/steps/eligibility_steps.py

```diff
@@ -306,16 +304,16 @@ class InputForm(SteuerlotseBaseForm):
                      ],
             validators=[InputRequired()])
 
-    class InputMultipleForm(SteuerlotseBaseForm):
-        alimony_eligibility = RadioField(
-            label="",
-            render_kw={'hide_label': True,
-                       'detail': {'title': _l('form.eligibility.alimony.detail.title'),
-                                  'text': _l('form.eligibility.alimony.detail.text')}},
-            choices=[('yes', _l('form.eligibility.alimony.multiple.yes')),
-                     ('no', _l('form.eligibility.alimony.multiple.no')),
-                     ],
-            validators=[InputRequired()])
+    def _pre_handle(self):
+        if self.number_of_users(self.stored_data) > 1:
+            self.form.alimony_eligibility.kwargs['choices'] = [('yes', _l('form.eligibility.alimony.multiple.yes')),
+                                                               ('no', _l('form.eligibility.alimony.multiple.no')),
+                                                               ]
+        else:
+            self.form.alimony_eligibility.kwargs['choices'] = [('yes', _l('form.eligibility.alimony.yes')),
+                                                               ('no', _l('form.eligibility.alimony.no')),
+                                                               ]
+        super()._pre_handle()
```

Comment:
To be honest, I did not think about `ngettext` because we had all those problems last time. That said, we could use it in the way that you are proposing with some smaller adjustments.